### PR TITLE
feat: multi issuer and deploy verify api blog post

### PR DESCRIPTION
**Pull request by Nebulis**

### CODE REVIEW

File: website/blog/2020-05-12-multi-issuers.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+title: Multi issuers and deploy verify API
+author: Laurent Maillet
+authorURL: https://twitter.com/Nebounet
+authorImageURL: https://lh3.googleusercontent.com/-kb0aLUOV4ek/AAAAAAAAAAI/AAAAAAAAAAA/AKF05nAdhdRMUzxqvrX-aSDCkiILApv-pQ.CMID/s192-c/photo.jpg
+---
+
+## Documentation organisation
+
+As part of our effort to ease the adoption of OpenCerts, we have worked on several items for the past weeks.
```
Comment (rjchow) [inline website/blog/2020-05-12-multi-issuers.md:10]:
```suggestion
As part of our efforts to ease the adoption of OpenCerts, we have worked on several items for the past weeks.
```

File: website/blog/2020-05-12-multi-issuers.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+### Multi issuers
+
+While having multiple issuers had always been available in OpenCerts, there were no explanation on how to do it, and the differences with having one issuer only. The use cases might be rare (for instance when there is a collaboration between multiple universities), but now [we got you covered](/docs/multi-issuer).
```
Comment (rjchow) [inline website/blog/2020-05-12-multi-issuers.md:14]:
```suggestion
While having multiple issuers has always been available in OpenCerts, there has been no documentation on this feature, as well how it differs from single issuer documents. The use cases might be rare (for instance when there is a collaboration between multiple universities), but now [we got you covered](/docs/multi-issuer).
```

File: website/blog/2020-05-12-multi-issuers.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+### Deploy verify API
+
+We have been provided an [API to verify OpenCerts document](https://github.com/OpenCerts/opencerts-functions) for quite a while, however we don't provide support or SLA for it. This led people to had to copy / paste the code and then deploy it into their own infrastructure.
```
Comment (rjchow) [inline website/blog/2020-05-12-multi-issuers.md:18]:
```suggestion
We have been providing an [API to verify OpenCerts document](https://github.com/OpenCerts/opencerts-functions) for quite awhile, however we don't provide support or SLA for it. This resulted in people copying and pasting the code and then deploying it into their own infrastructure.
```

File: website/blog/2020-05-12-multi-issuers.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+The code already provided scripts to automatically deploy to different infrastructure, but there were another problem. The repository is composed of multiple functions and while you might be interested by the API to verify document, you might not have been interested by the other APIs.
```
Comment (rjchow) [inline website/blog/2020-05-12-multi-issuers.md:20]:
```suggestion
The repository already contained scripts to automatically deploy to different infrastructure, but there was another complication. The repository contains multiple functions and while you might be interested by the API to verify documents, you might not be interested in the other APIs.
```

File: website/blog/2020-05-12-multi-issuers.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+We have worked on that and improved it:
+
+- every API is single-handedly deployable (including verify API)
```
Comment (rjchow) [inline website/blog/2020-05-12-multi-issuers.md:24]:
```suggestion
- every API is independently deployable (including verify API)
```

File: website/blog/2020-05-12-multi-issuers.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+- the process is [documented](/docs/api/verify#deployment)
+
+### Migration from v1 to v2
+
+When revamping the documentation, we forgot to bring back the one that covers the migration from v1 to v2 documentation. We fixed that problem, and [the documentation is available again](/docs/migrations/v1_to_v2)
```
Comment (rjchow) [inline website/blog/2020-05-12-multi-issuers.md:29]:
```suggestion
When revamping the documentation, we missed the article that covered the migration from v1 to v2. We fixed that problem, and [the documentation is available again](/docs/migrations/v1_to_v2)
```

File: website/blog/2020-05-12-multi-issuers.md [hunk 7]
```diff
@@ -0,0 +1,31 @@
+---
+title: Multi issuers and deploy verify API
+author: Laurent Maillet
+authorURL: https://twitter.com/Nebounet
+authorImageURL: https://lh3.googleusercontent.com/-kb0aLUOV4ek/AAAAAAAAAAI/AAAAAAAAAAA/AKF05nAdhdRMUzxqvrX-aSDCkiILApv-pQ.CMID/s192-c/photo.jpg
+---
+
+## Documentation organisation
```
Comment (rjchow) [inline website/blog/2020-05-12-multi-issuers.md:8]: title is weird

File: website/blog/2020-05-12-multi-issuers.md (hunk 2 above)
Comment (Nebulis) [reply website/blog/2020-05-12-multi-issuers.md:14]:
```suggestion
While having multiple issuers has always been available in OpenCerts, there has been no documentation on this feature, as well how it differs from single issuer documents. The use cases might be rare (for instance when there is a collaboration between multiple universities), but now [we got you covered](/docs/multi-issuer).
```

File: website/blog/2020-05-12-multi-issuers.md (hunk 7 above)
Comment (Nebulis) [reply website/blog/2020-05-12-multi-issuers.md:8]: indeed xD

**Pull request merged**